.github/workflows/release.yaml

```diff
@@ -351,6 +351,8 @@ jobs:
         with:
           base: main
           delete-branch: true
+          branch: create-pull-request/update-helm-index
```

Comment:
isn't the branch called [create-pull-request/patch](https://github.com/Dynatrace/dynatrace-operator/tree/create-pull-request/patch) ?

Reply:
correct, I don't like original branch name, I want to make it more clear for reader and for purpose of the PR.

Reply:
Ahhh i get it, thanks :+1: